implementation/tracker.m

```diff
@@ -492,7 +499,7 @@
             
             % Initialize the filter with zeros
             for k = 1:num_feature_blocks
-                hf{1,1,k} = zeros([filter_sz(k,1) (filter_sz(k,2)+1)/2 sample_dim(k)], 'like', params.data_type_complex);
+                hf{1,1,k} = complex(zeros([filter_sz(k,1) (filter_sz(k,2)+1)/2 sample_dim(k)]));
```

Comment:
1. `zeros(N, 'like', complex_data)` will return a **complex** matrix
1. `zeros(N, class(complex_data))` will return a **real** matrix
1. `complex(zeros(N, class(complex_data)))` is what we want(same as 1/)

```
>> N=2;
>> complex_data=complex(zeros(N, 'single'));
>>
>> zeros(N, 'like', complex_data)
ans =
  2×2 single matrix
   0.0000 + 0.0000i   0.0000 + 0.0000i
   0.0000 + 0.0000i   0.0000 + 0.0000i

>> zeros(N, class(complex_data))
ans =
  2×2 single matrix
     0     0
     0     0

>> complex(zeros(N, class(complex_data)))
ans =
  2×2 single matrix
   0.0000 + 0.0000i   0.0000 + 0.0000i
   0.0000 + 0.0000i   0.0000 + 0.0000i
```